README.md

```diff
@@ -90,16 +90,17 @@ export new Vuex.Store({
 
 It is possible to use vuex-persistedstate with Nuxt.js. It must be included as a NuxtJS plugin:
 
+#### With local storage (client-side only)
 ```javascript
 // nuxt.config.js
 
 ...
-plugins: [{ src: '~/plugins/localStorage.js', ssr: false }]
+plugins: [{ src: '~/plugins/persistedState.client.js']
```

Comment:
Why the explicit `persistedState.client.js` over `persistedState.js`? I guess that this can be confusing for readers, so lets keep it at `persistedState.js` to have both examples use the same filename.

Reply:
Ah ok its a Nuxt thing (I don't use Nuxt myself). Maybe add a comment + link about this name convention (https://nuxtjs.org/guide/plugins/#name-conventional-plugin).